docs/en/index.md

```diff
@@ -6,109 +6,109 @@
 
-# Collaboration
+So don't hesitate - join our **[Slack channel](https://statcan-aaw.slack.com/)** today and start getting the answers you need to succeed. We look forward to welcoming you to our community!
 
-There are many ways collaborate on the platform. Which is best for your
-situation depends on what you're sharing and how many people you want to share
-with. See the [Collaboration Overview](4-Collaboration/Overview.md) for details.
+Click on the link, then choose "Create an account" in the upper right-hand corner.
 
-Content to be shared breaks roughly into **Data**, **Code**, or **Compute
-Environments** (e.g.: sharing the same virtual machines) and who you want to
-share it with (**No one**, **My Team**, or **Everyone**). This leads to the
-following table of options
+!!! note ""
+    <center>
+    ![Use your @statcan.gc.ca email!](images/SlackAAW2.png)
+    <h3>Use your @statcan.gc.ca email address so that you will be automatically approved.</h3>
+    </center>
 
-|             |           **Private**            |                  **Team**                  |  **StatCan**  |
-| :---------: | :------------------------------: | :----------------------------------------: | :-----------: |
-|  **Code**   | GitLab/GitHub or personal folder |        GitLab/GitHub or team folder        | GitLab/GitHub |
-|  **Data**   |    Personal folder or bucket     | Team folder or bucket, or shared namespace | Shared Bucket |
-| **Compute** |        Personal namespace        |              Shared namespace              |      N/A      |
+## 🧭 Getting Started
 
-<!-- prettier-ignore -->
-??? question "What is the difference between a bucket and a folder?"
-    Buckets are like Network Storage. See the [Storage section](#storage) section for more discussion of the differences between these two ideas.
+To access AAW services, you need to log in to Kubeflow with your StatCan guest cloud account. Once logged in, select Notebook Servers and click the "New Server" button to get started.
+
+1. Login to [Kubeflow](https://kubeflow.aaw.cloud.statcan.ca/) with your StatCan guest cloud account. You will be prompted to authenticate the account.
+2. Select Notebook Servers.
+3. Click the "➕ New Server" button.
+
+## 🧰 Tools Offered
+
+AAW is a flexible platform for data analysis and machine learning. It offers a range of languages, including Python, R, and Julia. AAW also supports development environments such as VS Code, R Studio, and Jupyter Notebooks. Additionally, Linux virtual desktops are available for users who require additional tools such as OpenM++ and QGIS.
+
+Here's a list of tools we offer:
```

Comment:
I feel like we should mention somewhere: Bug reports + feat requests should be done via github/statcan/daaas